docs/gitops-install.md

```diff
@@ -62,6 +62,9 @@ UC_DEPLOY_EMAIL="my@email"
 The `UC_REPO` and `UC_DEPLOY` variables are local paths on your machine to where
 these two repos have been cloned.
 
+Optional variables of `UC_REPO_REF` and `UC_DEPLOY_REF` can be defined which
```

Comment:
You could probably add these two vars above in the example .env file, with "HEAD" as the default.